gateway-service/src/main/java/org/zowe/apiml/gateway/config/WebSecurity.java

```diff
@@ -155,8 +171,8 @@ private ResponseCookie createCookie(String name, String value) {
      * Security chain for oauth2 client. To enable this chain, please refer to Zowe OIDC configuration.
      */
     @Bean
-    @Order(Ordered.HIGHEST_PRECEDENCE)
-    public SecurityWebFilterChain oauth2WebFilterChain(
+    @Order(Ordered.HIGHEST_PRECEDENCE + 2)
```

Comment:
Probably, having a class with standard order values would help...

Reply:
This was a remanent of a test, I can remove this one, but yes, we should have a clear order, especially for the new code in the modulith cases.